.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
+<!--
+  Why is this change required? What problem does it solve?
+  This can be omitted if a linked issue is provided 
+-->
+
+## Description
+
+<!-- Describe your changes -->
+
+## Test Plan
```

Comment:
```suggestion
## Test and Deployment Plan
```

Reply:
I feel this sounds a bit too scary for contributors, refactored nonetheless https://github.com/coordinape/coordinape/pull/913/commits/02156d12d22bd4c7e27a498e24c8444a59c80b0f

Previously I had "How was this tested?" -- which sounds a bit more friendly, now it appears we need a bit more than "I ran this locally and see it working". What would you put in this section as the author?